package.json

```diff
@@ -3,31 +3,28 @@
   "version": "0.0.1",
   "private": true,
   "dependencies": {
-    "@capacitor/core": "1.3.0",
-    "@ionic/react": "^6.1.15",
-    "@ionic/react-router": "^6.1.15",
-    "@types/jest": "24.0.18",
-    "@types/node": "12.7.5",
-    "@types/react": "^16.9.43",
-    "@types/react-dom": "^16.9.8",
-    "@types/react-router": "^5.1.8",
-    "@types/react-router-dom": "^5.1.5",
+    "@capacitor/core": "^4.6.1",
+    "@capacitor/preferences": "^4.0.2",
+    "@ionic/react": "^7.0.0-rc.2",
+    "@ionic/react-router": "^7.0.0-rc.2",
     "date-fns": "^2.25.0",
     "ionicons": "^5.0.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "react-router": "^5.2.0",
```

Comment:
Can we also update the react router deps to the latest v5? I get the following peer dependency warning:

```
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: mini-create-react-context@0.4.1
npm WARN Found: react@18.2.0
npm WARN node_modules/react
npm WARN   react@"^18.2.0" from the root project
npm WARN   6 more (@ionic/react, @ionic/react-router, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@"^0.14.0 || ^15.0.0 || ^16.0.0 || ^17.0.0" from mini-create-react-context@0.4.1
npm WARN node_modules/react-router/node_modules/mini-create-react-context
npm WARN   mini-create-react-context@"^0.4.0" from react-router@5.2.0
npm WARN   node_modules/react-router
npm WARN 
npm WARN Conflicting peer dependency: react@17.0.2
npm WARN node_modules/react
npm WARN   peer react@"^0.14.0 || ^15.0.0 || ^16.0.0 || ^17.0.0" from mini-create-react-context@0.4.1
npm WARN   node_modules/react-router/node_modules/mini-create-react-context
npm WARN     mini-create-react-context@"^0.4.0" from react-router@5.2.0
```

(I had to fix the same issue in the swiper migration repo)